week1/project/ecommerce/src/App.js

```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react';
+import './App.css';
+import categories from './fake-data/all-categories';
+import products from './fake-data/all-products';
+import Categories from './components/Categories';
+import ProductsList from './components/Products';
+
+function App() {
+  const [state, setState] = useState({selectedCategory:'', isAll: true})
+  return (
+    <div className="App">
+      <h1>Products</h1>
+      { Categories({categories, state, setState}) }
+      { ProductsList({products, state}) }
+    </div>
```

Comment:
the components should be called like that
```js
<Categories categories={categories} state={state} setState={setState} />
 <ProductsList products={products} state={state} />
```

✏️need to modify✏️

Reply:
fixed